recipe-server/normandy/recipes/api/v2/views.py

```diff
@@ -203,3 +205,91 @@ def close(self, request, pk=None):
         approval_request = self.get_object()
         approval_request.close()
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class IdenticonView(views.APIView):
+    @staticmethod
+    def hex_luma(hex):
```

Comment:
@mythmon I think you mentioned something about `@staticmethod` when we discussed this previously? I don't remember exactly what we discussed however.

Reply:
The standard advice is to not use `@staticmethod`, but instead use a non-class function or `@classmethod`. Since you are not using anything from the class in this method, a non-class function would probably be appropriate.